HHS/Head-Start-TTADP
# [PROD] TTAHUB-2493: Prevent editing of objective title when linked to approved report

## Description of change

**TTAHUB-2493**
- Fix editing of objective title when goal is linked to an approved report.
- Fix adding of topics to a goal linked to an approved report.
- Fix adding resources to a goal linked to an approved report.

## How to test

- Edit a report using a goal linked to an approved report, you should not be able to edit the objective title.
- From the RTR add topics to a goal linked to an approved report.
- From the RTR add resources to a goal linked to an approved report. 

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-0


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
